tests/tr1/tests/algorithm/test.cpp

```diff
@@ -34,9 +34,9 @@ void test_single(char* first, char*) { // test single-element template functions
     CHECK_INT(STD minmax('0', '2', lessf).second, '2');
 
     char arr[] = {"012"};
-    STD initializer_list<char> ilist1(&arr[0], &arr[1]);
-    STD initializer_list<char> ilist2(&arr[1], &arr[3]);
-    STD initializer_list<char> ilist3(&arr[0], &arr[3]);
+    STD initializer_list<char> ilist1{arr[0]};
+    STD initializer_list<char> ilist2{arr[1], arr[2]};
+    STD initializer_list<char> ilist3{arr[0], arr[1], arr[2]};

```

Comment:
`arr` isn't used elsewhere, so this could be simplified by removing `arr` and replacing `arr[0]` with `'0'` etc.